### PR TITLE
Register form page

### PR DESCRIPTION
**Pull request by MichaelAlemayehu1**

### CODE REVIEW

File: client/lib/domain/auth/value_objects.dart [hunk 1]
```diff
@@ -37,16 +37,11 @@ class Name extends ValueObject<String> {
 }
 
 class Role extends ValueObject<String> {
-  static const List<String> roles = [
```
Comment (nati-m-afw) [inline client/lib/domain/auth/value_objects.dart:40]: @BarokDG already implemeted it in #25 . Choose one

File: client/lib/presentation/auth/widgets/sign_in_form.dart [hunk 2]
```diff
@@ -60,14 +60,18 @@ class SignInForm extends StatelessWidget {
                       .add(SignInFormEvent.passwordChanged(value)),
                   validator: (_) => BlocProvider.of<SignInFormBloc>(context)
                       .state
-                      .emailAddress
```
Comment (nati-m-afw) [inline client/lib/presentation/auth/widgets/sign_in_form.dart:63]: Nice catch 👍

**Review by nati-m-afw — APPROVED** [review]

File: client/lib/presentation/auth/widgets/sign_in_form.dart (hunk 2 above)
Comment (MichaelAlemayehu1) [reply client/lib/presentation/auth/widgets/sign_in_form.dart:63]: thanks

File: client/lib/domain/auth/value_objects.dart (hunk 1 above)
Comment (MichaelAlemayehu1) [reply client/lib/domain/auth/value_objects.dart:40]: yea but it kept saying assertion failed when i tried to run it

**Review by BarokDG — APPROVED** [review]

File: client/lib/domain/auth/value_objects.dart (hunk 1 above)
Comment (nati-m-afw) [reply client/lib/domain/auth/value_objects.dart:40]: you should prolly merge this and tell @BarokDG to delete his

Comment (MichaelAlemayehu1) [reply client/lib/domain/auth/value_objects.dart:40]: sure beka

**Pull request merged**